.gitignore

```diff
@@ -1,21 +1,42 @@
-.nyc_output
-coverage
-dist
-lerna-debug.log
+# Package manager
 node_modules
-npm-debug.log
-yarn-error.log
+
+# Build
+dist
 _webpackOutput
```

Comment:
Do you think `_webpackOutput` is still needed? What would generate this file? The only occurrence on Google for `_webpackOutput` is... this file.

Reply:
Hmm, good catch. I somehow just assumed it was something meaningful when I saw it was webpack related - but I can't find any external references to it anywhere either. 

From the git blame it seems to have been added by Ben 5 years ago

```bash
❯ grep -rnw cssnano -e '_webpackOutput'
(...)
cssnano/packages/cssnano/src/__tests__/_webpack.config.js:8:    path: path.resolve('./_webpackOutput/'),
```

Test related?

Reply:
You're right, it's a file that the tests create.